apps/prairielearn/assets/scripts/lib/qrCodeButton.ts

```diff
@@ -1,19 +1,26 @@
 import QR from 'qrcode-svg';
+import { observe } from 'selector-observer';
 
 import { onDocumentReady, parseHTMLElement } from '@prairielearn/browser-utils';
 
 onDocumentReady(() => {
```

Comment:
Nit: with the change to `observe`, `onDocumentReady` is no longer needed.

Reply:
Removed in a1bebd8.